DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -157,8 +164,16 @@ object BlobSinker extends SinkOperatorFactory {
     if(formatConf.isDefined && !formatConf.get.equalsIgnoreCase("json"))
       throw new Error(s"Output format: ${formatConf.get} as specified in the config is not supported")
     val outputFolders = blobOutputConf.groups.map{case(k,v)=>k->KeyVaultClient.resolveSecretIfAny(v.folder)}
-
-    val jsonSinkDelegate = (rowInfo: Row, rows: Seq[Row], outputPartitionTime: Timestamp, partitionId: Int, loggerSuffix: String) => {
+    
+	val sparkConf = ConfigManager.initSparkConf
+    val spark = SparkSessionSingleton.getInstance(sparkConf)
```

Comment:
we can just call directly SparkSessionSingleton.getInstance(ConfigManager.initSparkConf) instead of creating this variable sparkConf.

Reply:
Done